sklearn/feature_selection/_sequential.py

```diff
@@ -295,6 +300,18 @@ def fit(self, X, y=None, **params):
         self.support_ = current_mask
         self.n_features_to_select_ = self.support_.sum()
 
+        # Store the final cross-validation score of the selected feature set
+        X_new = X[:, self.support_]
+        self.final_cv_score_ = cross_val_score(
+            cloned_estimator,
+            X_new,
+            y,
+            cv=cv,
+            scoring=self.scoring,
+            n_jobs=self.n_jobs,
+            params=params if _routing_enabled() else None,
+        ).mean()
```

Comment:
This is adding quite a bit of compute time to `fit` time and the user cannot disable it. Ideally, we should only store things which we can compute as the process normally computes things, and if necessary, have a way to calculate this cv score if the user wants it.

I wouldn't be opposed to adding a method to this class which would run this cv and return the cv results along the mean instead of always computing it.